src/components/wallets/TopupWalletDialog.tsx

```diff
@@ -19,9 +19,7 @@ import { intlFormatNumber } from '~/core/intlFormatNumber'
 gql`
   mutation createCustomerWalletTransaction($input: CreateCustomerWalletTransactionInput!) {
     createCustomerWalletTransaction(input: $input) {
-      collection {
-        id
-      }
+      id
```

Comment:
Hum is that change related to this PR ? Also I feel it's wrong as I remember changing it to a collection last week 🤔

Reply:
yep I think the BE branch was not up to date then, checking with Vincent